CatalogIngestTool/Models.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace CatalogIngestTool;
+
+#nullable disable
```

Comment:
any reasons why we disable the nullable for the entities?

Reply:
Because they produce too many pointless warnings. This class is not an entity per se. It is used to de-serialize the JSON objects with from the import data.